src/PhotoPage.tsx

```diff
@@ -1,22 +1,111 @@
 import React from "react";
-import { Box, Typography } from "@mui/material";
+import { Box, ImageList, ImageListItem, Typography, useMediaQuery } from "@mui/material";
 
 import { MAIN_COLOR, SECONDARY_COLOR } from "./util";
+import { NavButton } from "./Shared/Button";
+
+const photos = [
+  {
+    img: './gallery/DSC07062.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/DSC07077.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/DSC07339.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/DSC07347.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/DSC07384.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/1.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/2.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/3.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/4.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/5.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/6.jpg',
+    alt: 'image1'
+  },
+  {
+    img: './gallery/7.jpg',
+    alt: 'image1'
+  },
+]
+
 
 export const PhotoPage = () => {
+  const isSmallScreen = useMediaQuery('(max-width:600px)');
+  const isMediumScreen = useMediaQuery('(max-width:900px)');
+  const isLargeScreen = useMediaQuery('(max-width:1200px)');
+  const getCols = () => {
+    if(isSmallScreen){
+      return 1;
+    } else if(isMediumScreen){
+      return 2;
+    } else if(isLargeScreen){
+      return 3;
+    } else{
```

Comment:
nit: put a space in between else and { 
```suggestion
    } else {
```